AstarteDeviceSDKCSharp/Data/AstarteFailedMessageStorage.cs

```diff
@@ -108,10 +156,29 @@ public async Task Reject(AstarteFailedMessageEntry failedMessages)
                 Trace.WriteLine($"Remove from local database "
                 + $"{failedMessages.GetTopic()} : "
                 + $"{failedMessages.GetPayload()}");
-                _astarteDbContext.AstarteFailedMessages.Remove(failedMessages);
+
+                if (sqliteDeleteConnection.State != ConnectionState.Open)
+                {
+                    sqliteDeleteConnection = new SqliteConnection(_dbConnectionString);
+                    sqliteDeleteConnection.Open();
+                }
+
+                if (sqliteDeleteConnection.State != ConnectionState.Open)
+                {
+                    sqliteDeleteConnection = new SqliteConnection(_dbConnectionString);
+                    sqliteDeleteConnection.Open();
+                }
```

Comment:
These state seems duplicated.
```suggestion
                if (sqliteDeleteConnection.State != ConnectionState.Open)
                {
                    sqliteDeleteConnection = new SqliteConnection(_dbConnectionString);
                    sqliteDeleteConnection.Open();
                }
```

Reply:
Removed duplicated code.